src/api/providers/fetchers/litellm.ts

```diff
@@ -32,11 +34,17 @@ export async function getLiteLLMModels(apiKey: string, baseUrl: string): Promise
 
 				if (!modelName || !modelInfo || !litellmModelName) continue
 
+				let determinedMaxTokens = modelInfo.max_tokens || modelInfo.max_output_tokens || 8192
+
+				if (modelName.includes("claude-3-7-sonnet")) {
+					// due to https://github.com/BerriAI/litellm/issues/8984 until proper extended thinking support is added
+					determinedMaxTokens = 64000
+				}
```

Comment:
Can we include the Anthropic header instead? https://docs.litellm.ai/docs/proxy/request_headers